drucker/drucker_dashboard_servicer.py

```diff
@@ -219,3 +224,73 @@ def UploadEvaluationData(self,
 
         return drucker_pb2.UploadEvaluationDataResponse(status=1,
                                                         message='Success: Uploading evaluation data.')
+
+    @error_handling(drucker_pb2.EvaluationResultResponse())
+    def EvaluationResult(self,
+                         request: drucker_pb2.EvaluationResultRequest,
+                         context: ServicerContext
+                         ) -> Iterator[drucker_pb2.EvaluationResultResponse]:
+        """ Return saved evaluation result
+        """
+        data_path = request.data_path
+        result_path = request.result_path
+        if not self.is_valid_upload_filename(data_path):
+            raise Exception(f'Error: Invalid evaluation file path specified -> {data_path}')
+        if not self.is_valid_upload_filename(result_path):
+            raise Exception(f'Error: Invalid evaluation result file path specified -> {result_path}')
+
+        eval_result_path = self.app.get_eval_path(result_path)
+        with open(eval_result_path + self.EVALUATE_DETAIL, 'rb') as f:
+            result_details = pickle.load(f)
```

Comment:
**Reminder**
At the moment, this implementation is OK.
But since this file may be large, we need to change the format to store.

Reply:
Could you make a **Issue** for this?

Reply:
@keigohtr 
I made https://github.com/rekcurd/drucker/issues/25 !
[UploadEvaluationData](https://github.com/rekcurd/drucker/blob/master/drucker/drucker_dashboard_servicer.py#L214-L218) has the same issue, so I also mentioned in the issue.